src/App.vue

```diff
@@ -42,83 +42,82 @@ const observable = require('./util/observable.js')
 const dbEnum = require(`../src-shared/db-enum.js`)
 const storage = require('./util/storage.js')
 
-window.addEventListener('message', (event) => {
+window.addEventListener(
+  'message',
+  (event) => {
     const eventData = event?.data?.eventData
     switch (event?.data?.eventId) {
       case 'theme':
-        window[rendApi.GLOBAL_SYMBOL_EXECUTE](rendApi.id.setDarkTheme, eventData.theme === 'dark')
+        window[rendApi.GLOBAL_SYMBOL_EXECUTE](
+          rendApi.id.setDarkTheme,
+          eventData.theme === 'dark'
+        )
         break
       case 'save':
         if (eventData.shouldSave) {
           window[rendApi.GLOBAL_SYMBOL_EXECUTE](rendApi.id.save)
         }
         break
     }
-  }, 
+  },
   false
 )
 
-async function initLoad(store) {
```

Comment:
This whole set of  changes seems counter-productive. It's not doing the RIGHT THING', it's only doing a different thing.